packages/google_sign_in/ios/Classes/GoogleSignInPlugin.m

```diff
@@ -56,6 +54,8 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [[GGLContext sharedInstance] configureWithError:&error];
     [GIDSignIn sharedInstance].scopes = call.arguments[@"scopes"];
     [GIDSignIn sharedInstance].hostedDomain = call.arguments[@"hostedDomain"];
+    UIViewController *viewController = [UIApplication sharedApplication].keyWindow.rootViewController;
+    [GIDSignIn sharedInstance].uiDelegate = viewController;
```

Comment:
Our view controller doesn't implement the `GIDSignInUIDelegate` protocol (https://developers.google.com/identity/sign-in/ios/api/protocol_g_i_d_sign_in_u_i_delegate-p)

So at best, isn't that assignment a no-op, and and at worst, it's just plain wrong?